testing/libmustache/APKBUILD

```diff
@@ -0,0 +1,52 @@
+# vim: smartindent noexpandtab tabstop=4 shiftwidth=4
+# Maintainer: Renoir Boulanger <hello@renoirboulanger.com>
+
+pkgname=libmustache
+_pkghash=c809212c2941f5aa8030ce791efa38a22f43af62
+pkgver=0.4.2
+#_urlver=v${pkgver}
+_urlver=${_pkghash}
+#_pkgver=${pkgver}
+_pkgver=${_pkghash}
```

Comment:
I'm a bit confused by the different variables for the same thing -- could `_pkghash` be simply called `_gitrev` and applied everywhere?